packages/ui/src/styles/utilities/_variables.scss

```diff
@@ -22,19 +22,31 @@ $danger-light: #FCEBE9;
 
 $light: #f6f6f6;
 
-$theme-colors: map-merge($theme-colors, (
-        'primary': $primary,
+$base-colors: (
+  'primary': $primary,
+  'secondary': $secondary,
+  'success': $success,
+  'warning': $warning,
+  'danger': $danger,
+  'light': $light,
+  'info': $info
+);
+
+$badge-font-size: 0.75rem;
+
+$subtle-colors: (
+        'primary-subtle': $primary-light,
```

Comment:
Isnt this more RSD stuff?

Reply:
I don’t think because we have light and dark variants in forge as well, I just decided to split them up so that we’re not generating as much css by cycling through all of the variants